### PR TITLE
Update Denmark Marriage Abroad Smart Answer 

### PR DESCRIPTION
**Pull request by 1pretz1**
Removes two questions:

- What is your partner’s nationality?
- Is your partner of the opposite sex, or the same sex?

The outcomes have now been consolidated into three, as the above questions had no impact on the information the user received. 

Trello:
https://trello.com/c/FpoDkaVp/1454-marriage-abroad-denmark-remove-questions-and-outcomes

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/flows/marriage_abroad_flow/outcomes/countries/denmark/_ceremony_country.erb [hunk 1]
```diff
@@ -1,27 +1,31 @@
+<% # TBC %>
```
Comment (edwardkerry) [inline app/flows/marriage_abroad_flow/outcomes/countries/denmark/_ceremony_country.erb:1]: Can we take these out, from all 3 new ERBs?

**Review by edwardkerry — COMMENTED** [review]
This looks good to me, though I think some comments have been left in.

I have rebased on main due to merge conflicts (changes in the now deleted files) but otherwise not touched this work.

File: app/flows/marriage_abroad_flow/outcomes/countries/denmark/_ceremony_country.erb (hunk 1 above)
Comment (1pretz1) [reply app/flows/marriage_abroad_flow/outcomes/countries/denmark/_ceremony_country.erb:1]: Yep, I originally left them in to make it obvious that the wrapped content was yet to be confirmed, however the changes have been approved so TBC lines can now be removed. I'll do this now.

**Comment by 1pretz1** [comment]
Thanks for the review @edwardkerry - I've removed the TBC comments as the changes have now been fact checked. Thanks for rebasing and resolving those conflicts. Could you have another gander?

**Review by edwardkerry — APPROVED** [review]
All looks good to me!

**Pull request merged**